app/auth/middleware.test.js

```diff
@@ -1,13 +1,13 @@
 const proxyquire = require('proxyquire')
 
-const referenceDataService = require('../../common/services/reference-data')
-
-function UserStub(token) {
-  this.user_name = token.user_name
+const locationStub = {
+  getUserLocations: token => {
+    return ['TEST']
+  },
 }
 
 const authentication = proxyquire('./middleware', {
-  '../../common/lib/user': UserStub,
+  '../../common/services/user-locations': locationStub,
```

Comment:
This looks like we're not longer stubbing the User class. Why has this been removed?

Reply:
It's no longer necessary since the `User` class is now much simpler and has no external dependencies.

Reply:
I'm a little uncomfortable removing this stub as if the User library changes in the future and gets more complex it will be being pulled into this test and could affect it.

I think it's good to stub services to what you expect in the tests rather than allow it to use all the dependencies.